docs/telegram-integration-flow.md

```
@@ -0,0 +1,172 @@
+
+-- Ключи идемпотентности для анонимных пользователей
+processed_idempotency_keys (
+  id, telegram_chat_id, idempotency_key, message_id, created_at
+)
+```
+
+## Флоу обработки сообщений
+
+### 1. Анонимные пользователи
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)